lib/attach-middleware.js

```diff
@@ -12,12 +12,12 @@ function logError(err, req, res, next) {
   next(err);
 }
 
-module.exports = function(app, options) {
+module.exports = function(app, project) {
```

Comment:
I think this should still take options and get a configPath and root as properties